linux-vm-with-cache/azuredeploy.json

```diff
@@ -0,0 +1,230 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
```

Comment:
Remove default location and add allowed values like in https://github.com/Azure/azure-quickstart-templates/blob/master/101-create-availability-set/azuredeploy.json